.github/actions/deploy/action.yml

```diff
@@ -11,9 +11,9 @@ inputs:
   docs-output-prefix:
     required: true
     description: "Path prefix where the docs will be generated"
-  docs-workflow-token:
+  github-token:
```

Comment:
I renamed this input, because it's not actually specific to the docs generation, it should also be used by the deployment script.